Sources/KSCrashRecording/KSCrashReportStore.c

```diff
@@ -160,18 +158,14 @@ static void initializeIDs(void)
     time(&rawTime);
     struct tm time;
     gmtime_r(&rawTime, &time);
-    int64_t baseID = (int64_t)time.tm_sec
-                   + (int64_t)time.tm_min * 61
-                   + (int64_t)time.tm_hour * 61 * 60
-                   + (int64_t)time.tm_yday * 61 * 60 * 24
-                   + (int64_t)time.tm_year * 61 * 60 * 24 * 366;
+    int64_t baseID = (int64_t)time.tm_sec + (int64_t)time.tm_min * 61 + (int64_t)time.tm_hour * 61 * 60 +
+                     (int64_t)time.tm_yday * 61 * 60 * 24 + (int64_t)time.tm_year * 61 * 60 * 24 * 366;
```

Comment:
nit: this is less readable now 😞

Reply:
Maybe disable formatter in such portions of code that are manually formatted then?

Reply:
Yeah, would be great to have some places with disabled formatter as an example.
And by the way, would be great to add contributing guidelines (as a separate PR sure)